client/src/components/Visualization/Visualization.css

```diff
@@ -0,0 +1,36 @@
+.visualization-container {
+  width: 100%;
+}
+
+.chart {
+  margin-bottom: 50px;
+}
+
+select {
+  border-radius: 5px;
+  color: #727272;
+  font-size: 16px;
+  margin-bottom: 2em;
+  outline: none;
+  padding: 7px 5px;
+  width: 200px;
```

Comment:
Not sure what this css is for, but if it's for a chart type or something consider changing `px` to something responsive.